ggplot/qplot.py

```diff
@@ -69,8 +67,8 @@ def qplot(x, y=None, color=None, size=None, fill=None, data=None,
     >>> print qplot('mpg', data=mtcars, geom="hist", main="hist")
     >>> print qplot('mpg', data=mtcars, geom="histogram", main="histogram")
     >>> print qplot('cyl', 'mpg', data=mtcars, geom="bar", main="bar")
-    >>> print qplot('mpg', 'drat', data=mtcars, xlab= "x lab", main="xlab")
-    >>> print qplot('mpg', 'drat', data=mtcars, ylab = "y lab", main="ylab")
+    >>> print qplot('mpg', 'drat', data=mtcars, xlabel= "x lab", main="xlab")
```

Comment:
I know these are just examples and maybe i'm being too naggy here but we should decide if we want spaces before and after the `=`. Its too small an issue so we can leave it as it is. Thanks :)

Reply:
No, PEP8 is important.  This goes to show that I'm over-reliant on my linter.  Should be fixed now.

Thanks!